packages/vscode-extension/src/utilities/common.ts

```diff
@@ -16,16 +16,24 @@ export function getDevServerScriptUrl() {
 
 export function isWorkspaceRoot(dir: string) {
   const packageJsonPath = path.join(dir, "package.json");
-  let workspaces;
+  const nxJsonPath = path.join(dir, "nx.json");
+
   try {
-    workspaces = require(packageJsonPath).workspaces;
+    require(nxJsonPath);
```

Comment:
Is there a scenario in which require fails for reasons unrelated to nx.json existence?
Could we use `fs.existSync()` here?

Reply:
Changed in: https://github.com/software-mansion/radon-ide/pull/724/commits/7264f94c25a04bb0854ce8bd2beb27333d41a839